src/components/LeftSidebar/NewGroupConversation/NewGroupConversation.vue

```diff
@@ -265,6 +285,15 @@ export default {
 				this.passwordProtect = false
 			}
 		},
+
+		success(value) {
```

Comment:
Let's also add the same, but watch for `error` and focus `Close` button instead, as it'll be only available